css/style.css

```diff
@@ -0,0 +1,269 @@
+.body {
+  max-width: 1440px;
+  margin: 0 auto;
+  font-family: "Montserrat", sans-serif;
+  color: #000;
+}
+
+.header {
+  padding: 24px 60px 0 42.72px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__img {
+  display: block;
+}
+
+.header__menu_link {
+  display: block;
+  color: #000;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  text-decoration: none;
+  line-height: 18px;
+  transition-duration: .2s, .2s;
+  transition-property: color, text-decoration;
+}
+
+.header__menu_link:hover{
+  color: #e51937;
+  text-decoration: underline;
+}
+
+.header__menu {
+  display: flex;
+  justify-content: space-around;
+}
+
+.header__menu_link-red {
+  color: #e51937;
+  text-decoration: underline;
+  text-underline-position: under;
+  transition-duration: .2s;
+}
+
+.header__menu_link-red:hover{
+  font-size: 20px;
+}
+
+.bloks-1 {
+  flex-grow: .05;
+}
+
+.bloks-2 {
+width: 39%;
+}
+
+.bloks-3 {
+  width: 15%;
+}
+
+.link__item {
+  display: block;
+  transition-duration: .2s, .2s;
+  transition-property: color, text-decoration;
+}
+
+.link__item:hover{
+  color: #e51937;
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.link__img {
+  width: 24px;
+  height: 15px;
+  transition-duration: .3s, .3s;
+  transition-property: widht, height;
+}
+
+.link__img:hover{
+  width: 34px;
+  height: 25px;
+  cursor: pointer;
+}
+
+.link {
+  display: flex;
+  justify-content: space-between;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+}
+
+.car__title {
+  text-align: center;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 90px;
+  line-height: 110px;
+  margin-bottom: 0;
+  margin-top: 60px;
+}
+
+.car__subtitle {
+  position: relative;
+  bottom: 11px;
+  text-align: center;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 15px;
+  margin-top: 0;
+  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)